qiita_db/util.py

```diff
@@ -714,7 +714,17 @@ def path_builder(db_dir, filepath, mountpoint, subdirectory, obj_id):
                 for fpid, fp, fp_type_, m, s in results]
 
 
-def purge_filepaths():
+def _rm_files(TRN, fp):
```

Comment:
I don't think you need to pass TRN as a parameter, given that it is a global variable in the system, you can just use it.